Makefile

```diff
@@ -159,16 +162,46 @@ add-private-key:
 	docker exec issuer-vault-1 \
 	vault write iden3/import/pbkey key_type=ethereum private_key=$(private_key)
 
-# usage: make private_key=xxx add-private-key-localstorage
-.PHONY: add-private-key-localstorage
-add-private-key-localstorage:
-	docker exec issuer-api-1 \
-	./kms_local_storage_priv_key_importer --privateKey=$(private_key)
+## Usage:
+## AWS: make private_key=XXX aws_access_key=YYY aws_secret_key=ZZZ aws_region=your-region import-private-key-to-kms
+## localstorage and vault: make private_key=XXX import-private-key-to-kms
+.PHONY: import-private-key-to-kms
+import-private-key-to-kms:
+ifeq ($(ISSUER_KMS_ETH_PROVIDER), aws)
+	@echo "AWS"
+	docker build --build-arg ISSUER_KMS_ETH_PROVIDER_AWS_ACCESS_KEY=$(aws_access_key) \
+    		  --build-arg ISSUER_KMS_ETH_PROVIDER_AWS_SECRET_KEY=$(aws_secret_key) \
+    		  --build-arg ISSUER_KMS_ETH_PROVIDER_AWS_REGION=$(aws_region) -t privadoid-kms-importer -f ./Dockerfile-kms-importer .
+	$(eval result = $(shell docker run -it -v ./.env-issuer:/.env-issuer -v $(ISSUER_KMS_PROVIDER_LOCAL_STORAGE_FILE_PATH)/kms_localstorage_keys.json:/localstoragekeys/kms_localstorage_keys.json \
```

Comment:
Why do we need to pass the path `/localstoragekeys/kms_localstorage_keys.json` to localstorage_key if we import a key to AWS?